src/vs/workbench/browser/parts/activitybar/activitybarActions.ts

```diff
@@ -628,6 +628,15 @@ registerAction2(
 
 registerThemingParticipant((theme, collector) => {
 
+	const activityBarActiveBorderColor = theme.getColor(ACTIVITY_BAR_ACTIVE_BORDER);
```

Comment:
@aeschli why exactly can this not be expressed via CSS?

Reply:
I do not fully understand it but the dom node containing the icon (a :`before`) does not see the css color variable definitions.